loadJsonFile.py

```diff
@@ -0,0 +1,51 @@
+import json
+import requests
+
+with open('result.json') as f:
```

Comment:
Can we make the variable name more meaningful? If 'f' can be another word that actually conveys meaning, that'd be great

Reply:
Ok, I check now!